### PR TITLE
swagger implemented in image feature

### PR DESCRIPTION
**Pull request by JuanCarlosMartinez4**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java [hunk 1]
```diff
@@ -82,7 +87,9 @@
      * @param request contains client request data.
      * @return Response it mean the result of the conversion.
      */
-    @PostMapping()
+    @PostMapping("/imageConverter")
+    @ApiOperation(value = "Image specifications", notes = "provide values for converting Image file to other one",
```
Comment (Enrique-C) [inline src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java:91]: Use magick number here please

Comment (JuanCarlosMartinez4) [reply src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java:91]: in the comment ?

Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java:91]: No, in code line 91 "provide ...."

Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java:91]: If it is only a note, you can fix it in other feature

**Pull request merged**